docs/leo-rover/manuals/boot-and-firmware.mdx

```diff
@@ -19,26 +16,32 @@ Prepare:
   height="802"
 />
 
-To connect to Leo Rover, see:
+---
+
+## Step 2
 
-<LinkButton docId="leo-rover/guides/connect-to-rover-ap" />
+<h3>Connect to Leo Rover by following this tutorial:</h3>
 
-:::note
+## <LinkButton docId="leo-rover/guides/connect-to-rover-ap" />
 
-This time you won't see the battery LED flashing. It will work only after
-successfully performed Step 2.
+---
 
-:::
+## Step 3
 
-## Step 2: Flash the firmware
+<h3>Flash the new firmware by referring to:</h3>
 
-Refer to:
+## <LinkButton docId="leo-rover/guides/firmware-update" />
 
-<LinkButton docId="leo-rover/guides/firmware-update" />
+---
 
-## Step 3: Access the Leo Rover UI
+## Step 4
 
-Open any web browser and type:
+<h3>
+  The rover should be fully functional now.\ To test it access the Leo Rover UI
+  and drive a little.
+</h3>
```

Comment:
Prettier does not seem to work with `\n` line breaks inside a header. Maybe instead of a header, a `success` note would be more appropriate?

```suggestion
:::success

The rover should be fully functional now. To test it, access the Leo Rover UI
and drive a little.

:::
```

Reply:
Done + text rearranged to fit better on the site.